api/src/main/java/com/mainstreethub/project/api/v1/CreateUserRequest.java

```diff
@@ -1,4 +1,4 @@
-package com.mainstreethub.project;
+package com.mainstreethub.project.api.v1;
```

Comment:
Not really sure we want to advocate sandboxing stuff into a package like this.  You force either copying/pasting of classes that don't change between API versions into new packages or a really confusing to use API for users.

Version changes should ideally happen quickly, so I'm not sure a package structure is warranted.  If we have a plan for one that we know will happen slowly then it probably makes sense, but not sure the base template should encourage the slow moving process by default.